### ISSUE TITLE
Moving towards MDC (MDL v2.0)

### ISSUE THREAD

**Issue by Zodiase** [Zodiase/meteor-mdl#59]
I'm testing out how well MDC ([Material Components](https://github.com/material-components/material-components-web)) work with Blaze and Meteor and see if a Meteor package could help.

**Comment by Zodiase** [comment]
# Experiments (keep updated):
1. Use MDC from npm directly in a Meteor app (using release v1.4.2.3).
    1. [x] Import bundled distribution js and css files in app.
    See test case: https://github.com/Zodiase/Meteor-Tests/tree/try-material-components--distribution
    2. [x] Import module distribution js and css files in app.
    See test case: https://github.com/Zodiase/Meteor-Tests/tree/try-material-components--module-distribution
    3. [ ] Import module source js and sass files in app.
        1. Error when using sass files: Zodiase/meteor-mdc/issues/1
2. Use MDC from a Meteor package in a Meteor app (using release v1.4.2.3).
    1. [x] Import mdc bundle, without any styling, from the package. (The styles are still imported from the NPM package)
    See test case: https://github.com/Zodiase/Meteor-Tests/tree/f8009b26cb437f7a7c0f749d3b8a9c8b2f233bb8
    Meteor Package: https://atmospherejs.com/zodiase/mdc-styleless

**Comment by Zodiase** [comment]
Note: when `zodiase:mdc` is stable, catches up with NPM MDC package's version and reaches `1.0.0`, `zodiase:mdl@2.0.0` will be released and be pointed to `zodiase:mdc@1.0.0` (using `api.imply`).

**Comment by davidho8087** [comment]
zodiase:mdl@2.0.0 released?

**Comment by Zodiase** [comment]
@1073113 No, not yet. The official MDC (Material Components) is currently still at version `0.7.0` which means things are still up to change.
